### PR TITLE
Bump grafana from 8.3.1 to 8.3.4

### PR DESCRIPTION
**Pull request by jovial**
Bump the grafana version to 8.3.4 from 8.3.1 to include the fix for the following issue:

https://github.com/grafana/grafana/issues/42605

We seem to be using a variable for the min-step in a number of our dashboards.

### CODE REVIEW

**Comment by markgoddard** [comment]
LGTM, I'll build an image with this repo.

**Comment by jovial** [comment]
> LGTM, I'll build an image with this repo.

Thank you.

**Comment by markgoddard** [comment]
Will be tagged as `wallaby-20220119T122428`

**Comment by jovial** [comment]
> wallaby-20220119T122428

Should I update this PR with that version? Or do you want a separate PR for that change?

**Comment by markgoddard** [comment]
> > wallaby-20220119T122428
> 
> Should I update this PR with that version? Or do you want a separate PR for that change?

Please update this PR.

**Comment by markgoddard** [comment]
@jovial I'll let you promote the container image tag: https://github.com/stackhpc/stackhpc-release-train/actions/workflows/container-promote.yml

It will be backed up behind a sync job for a while. We could cancel that if you need this sooner.

**Comment by jovial** [comment]
Suggested edit:
```diff
diff --git a/etc/kayobe/kolla/globals.yml b/etc/kayobe/kolla/globals.yml
--- etc/kayobe/kolla/globals.yml
+++ etc/kayobe/kolla/globals.yml
@@ -2,8 +2,9 @@
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
 
 cloudkitty_tag: wallaby-20220119T122428
+grafana_tag: wallaby-20220119T122428
 horizon_tag: wallaby-20220119T122428
 manila_tag: wallaby-20211210T140839
 neutron_tag: wallaby-20220104T102739
 neutron_sriov_agent_tag: wallaby-20220110T113538

```

File: etc/kayobe/kolla/globals.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_facts.distribution | lower }}{% endraw %}/gpg"
 
 cloudkitty_tag: wallaby-20220119T122428
+grafana_tag: wallaby-20220119T122428
```
Comment (markgoddard) [inline etc/kayobe/kolla/globals.yml]: Wrong tag!

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/kolla/globals.yml (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/kolla/globals.yml]: Sorry, my mistake. I grabbed the tag too early. It should be `wallaby-20220210T160332`.

**Review by markgoddard — APPROVED** [review]

**Comment by jovial** [comment]
Tested and seems to be working. Thanks Mark.

**Pull request merged**